routes/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+
+const createWithShop = require('../middleware/withShop');
+const createShopifyAuthRouter = require('./shopifyAuth');
+const shopifyApiProxy = require('./shopifyApiProxy');
+
+module.exports = function createRouter(shopifyConfig) {
+  const router = express.Router();
+
+  router.use('/auth/shopify', createShopifyAuthRouter(shopifyConfig));
+  router.use('/api', createWithShop({ redirect: false }), shopifyApiProxy);
```

Comment:
could this use withShop imported from the middleware index instead? I realize they are the same thing but this would be more consistent with how we want others to use it.

Reply:
The one from the middleware index is already generated with `redirect: true`. We could go back to exporting the factory function instead of the middleware itself but that would make the two of them weirdly different to consumers.